### DISCUSSION TITLE
How to reverse array in JavaScript?

### DISCUSSION THREAD

**Discussion by PhantomGuru0427** [community/community#20100] [Codespaces]
This is my javascript code.
I need to reverse array in this code.

const array = ["car", "tank", "port", "airplane"].

const reverse_array = array.map((array) => {return array}).
contact me, please.
Thank you.

**Comment by lucasvm8** [comment]
Hello.
You can use only reverse too, without map.

const arr = [1, 2,3]

arr.reverser()

outuput = [3. 2. 1]

**Comment by edward795** [comment]
**Note:**
**The reversal is condition specific:**

1. You want to reverse the original array elements (mutate):
      `arr.reverse()`
2.You don't want to reverse the original array & just want to get a copy of the array with reversed elements (not mutating the original array):
     `const revArr=[...arr].reverse()            --> use ES6 spread operation`

**Reply by edward795** [reply]
**Imp**

Stick to ES6 syntax & features as these are updated industry standards & will show much more professionalism.You can refer my repo for some of the ES6 next gen javascript features.

https://github.com/edward795/Next_Generation_Javascript-ES6-Features
